controllers/ncnews.controller.js

```diff
@@ -47,3 +51,35 @@ exports.getArticleComments = (req, res, next) => {
         next(err)
     })
 }
+
+exports.postCommentToArticle = (req, res, next) => {
+    
+    const newComment = req.body
+    const {article_id} = req.params
+
+    //Q7 refactoring to do - articleExistenceQuery below is the start of using Promise.all for advanced error handling
+    // const articleExistenceQuery = checkArticleExists(article_id)  
+
+
+
+    insertArticleComment(newComment, article_id)
+    .then(({rows})=>{
+        const postedComment = rows[0]
```

Comment:
Convention would be to return rows[0] from the model so no need to access the object here.